Remora.Discord.Extensions/Formatting/Markdown.Timestamps.cs

```diff
@@ -0,0 +1,89 @@
+//
+//  Markdown.Timestamps.cs
+//
+//  Author:
+//       Jarl Gullberg <jarl.gullberg@gmail.com>
+//
+//  Copyright (c) 2017 Jarl Gullberg
+//
+//  This program is free software: you can redistribute it and/or modify
+//  it under the terms of the GNU Lesser General Public License as published by
+//  the Free Software Foundation, either version 3 of the License, or
+//  (at your option) any later version.
+//
+//  This program is distributed in the hope that it will be useful,
+//  but WITHOUT ANY WARRANTY; without even the implied warranty of
+//  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+//  GNU Lesser General Public License for more details.
+//
+//  You should have received a copy of the GNU Lesser General Public License
+//  along with this program.  If not, see <http://www.gnu.org/licenses/>.
+//
+
+using System;
+
+namespace Remora.Discord.Extensions.Formatting
+{
+    public partial class Markdown
+    {
+        /// <summary>
+        /// Formats a Unix timestamp value into Discord Markdown Timestamp.
+        /// </summary>
+        /// <param name="unixTimestamp">The Unix timestamp to format.</param>
+        /// <param name="timestampStyle">The style to format into.</param>
+        /// <returns>
+        /// A Discord markdown-formatted Timestamp string.
+        /// </returns>
+        public static string Timestamp(long unixTimestamp, TimestampStyle? timestampStyle = default)
```

Comment:
Instead of making this nullable, this should default to short date and time
(:f) as it is the default in the client.

Reply:
This was how it was originally, but Jax suggested that it be done this way instead.

Reply:
My original reasoning was that since Discord allows you to omit the style specifier, having no specifier as our default would be more appropriate. This will - at least to my knowledge - shift the responsibility of deciding what the default is to Discord, leaving us more flexible if they decide to change their minds later.